base/dict.jl

```diff
@@ -686,6 +686,20 @@ end
 
 filter!(f, d::Dict) = filter_in_one_pass!(f, d)
 
+function map!(f, iter::ValueIterator{Dict{K,V}}) where {K, V}
```

Comment:
Since you don't use `K` and `V` you can just declare `iter::ValueIterator{<:Dict}`